### ISSUE TITLE
OED support for agriculture

### ISSUE THREAD

**Issue by stufraser1** [OasisLMF/OpenDataStandards#76]
OED currently supports Agriculture, Greenhouse, Forestry (Unknown, Spruce, Birch, Pine).
Development / Humanitarian sector application of risk models often requires a more detailed breakdown of agriculture into crop types and livestock types. 
FAO provides a taxonomy for these, and this has been implemented by GFDRR in the Risk Data LIbrary Standard. To assist dev/hum sector use of OED and LMF, and interoperability with RDL, it is proposed to expand the support for agri and forestry.

Detailed typology is proposed to be added via new construction codes, making use of the existing occupancy codes, potential with some amendment to their description. Livestock would be require a new occupancy code.

Attached - mapping of FAO taxonomy to proposed OED codes: 
[OED_LocInfrastructure_Agriculture.xlsx](https://github.com/OasisLMF/OpenDataStandards/files/9027600/OED_LocInfrastructure_Agriculture.xlsx)

**Comment by MattDonovan82** [comment]
@stufraser1 The suggestions for new constructions to support crops, livestock and forestry look reasonable to me.

My only concern (which you have alluded to) is your suggestion of amending current occupancy codes. Users may already be using 1351 for their "general" agriculture which as it states covers multiple uses and not just crop (i.e. soil prep, veterinary, farm labour etc). My suggestions would be to add new occupancy codes as follows:

1360 -  agriculture - crops
1370 - livestock

In regards to forestry - this is a little bit more complicated just because specific options (spruce, birch and pine) already exist and simply adding another "forestry" next to "forestry, unknown" might be ambiguous.

I think your suggestion of changing the current "Forestry, Unknown" to "Forestry" or perhaps even "forestry, general" might be the better option?

Any other suggestions @johcarter @jones-matt @benhayes21

**Comment by MattDonovan82** [comment]
anymore thoughts on this @stufraser1 ?

**Comment by stufraser1** [comment]
I support addition of 
1360 - agriculture - crops
1370 - livestock

I support changing the current "Forestry, Unknown" to "Forestry" rather than "forestry, general".

**Comment by jones-matt** [comment]
I agree with suggestion to add in 1360 and 1370 and leaving 1353 as is (or modifying the description to Forestry, General)

Moving 1354, 1355 and 1356 logically makes sense, but will be used by existing vendors. We don't want duplication so I think moving these to construction means removing them from Occupancy but that would be a breaking change with would need SG approval.

Adding in a new section of 8000 codes is quite a large addition to OED. I think such a substantial addition should go to SG for approval anyhow - even though it's not breaking. There may be other agricultural codesets already used that need consideration.

**Comment by MattDonovan82** [comment]
**NOTE:** All changes here will go into the next major release v3.0.0 at the end of this year so will need SC signoff. As per the ODS governance signoff process, there will be 3 months notice given to the community from the 'release candidate' to the actual release of ODS v3.0.0. This should provide enough time for users to perform any UAT and make the required changes.

If we are adding 1360 and 1370 as new occ codes for agriculture, maybe we should amend the current description for occupancy code 1351 from "Agriculture" to "Agriculture, general" ? Just to be more explicit?

I agree moving the (spruce, birch and pine) from occupancy to construction within the new 8000 codes makes sense. Can you suggest the new construction code and description for these three @stufraser1

More comments welcome

**Comment by MattDonovan82** [comment]
@stufraser1 should the new construction codes for (spruce, birch and pine) be the same as the current occ codes? Any other additional comments?

**Comment by MattDonovan82** [comment]
Closed as included in OED v3.0.0
